test/groups/array_manifold.jl

```diff
@@ -3,7 +3,7 @@
     M = Rotations(3)
     AG = ArrayManifold(G)
     @test base_group(AG) === G
-    @test Manifolds.is_decorator_group(AG) === Val(true)
```

Comment:
I'd leave the test for `@test Manifolds.val_is_decorator_group(AG) === Val(true)`.